.Lib9c.Tools/SubCommand/Tx.cs

```diff
@@ -88,7 +88,7 @@ public class Tx
                         nameof(Nekoyume.Action.MigrationActivatedAccountsState) => new MigrationActivatedAccountsState(),
                         nameof(Nekoyume.Action.MigrationAvatarState) => new MigrationAvatarState(),
                         nameof(Nekoyume.Action.CreatePendingActivations) => new CreatePendingActivations(),
-                        nameof(Nekoyume.Action.RenewAdminState) => new RenewAdminState(),
+                        nameof(Nekoyume.Action.RenewAdminState0) => new RenewAdminState0(),
```

Comment:
I guess we need to keep a latest action name

Reply:
Thank you for letting me know it.

```
$ dotnet run -p .Lib9c.Tools -- tx renew-admin-state 100
6c7531353a52656e657741646d696e53746174656475323a696431363a424a1d1fc238984797fc51e8b0b2c6817531353a6e65775f76616c69645f756e74696c69313030656565
```

Reply:
I amended it in ad6b757 commit.